docs/README.md

```
@@ -0,0 +1,48 @@
+# Rundown Documentation
+
+This index routes you to the right document based on what you're trying to do.
+
+---
+
+## I want to...
+
+### Write or run runbooks
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)